### ISSUE TITLE
'twisted.application.strports' has no attribute 'listen' on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8789]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8789|
|Type|enhancement|
|Created|2016-08-27 14:15:46Z|
|Branch|https://github.com/twisted/twisted/tree/8789-rodrigc-strports|

While trying to port buildbot to use Python 3, I ran across this error in Twisted:

  File "<stdin>", line 1
    AttributeError: module 'twisted.application.strports' has no attribute 'listen'
                  ^

buildbot uses the twisted.application.strports.listen() method

<details><summary>Searchable metadata</summary>

```
trac-id__8789 8789
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8789_rodrigc_strports 8789-rodrigc-strports
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1472307346537642 1472307346537642
changetime__1472327697992280 1472327697992280
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/528

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

changes looks good. Please merge

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset c712e86b89cb94b7ca7c7bd482cd6afa0be3a43a
```
#!CommitTicketReference repository="" revision="c712e86b89cb94b7ca7c7bd482cd6afa0be3a43a"
Merge pull request #528 from twisted/8789-rodrigc-strports

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8789

Make listen() and parse() visible symbols on Python 3
```
